### PR TITLE
Add cozystack-cluster-admin

### PR DESCRIPTION
**Pull request by klinch0**
## Summary by CodeRabbit

- **New Features**
	- Introduced new `Secret` resources for `k8s-client`, `kubeapps-client`, and `kubeapps-auth-config` to enhance Keycloak configuration.
	- Added a new `KeycloakRealmGroup` named `cozystack-cluster-admin` for improved access management.
	- Implemented a new `RoleBinding` for `kubeapps-admin` in the `cozy-public` namespace, linking it to the `kubeapps-admin` role.
	- Created a new `ClusterRoleBinding` named `cozystack-cluster-admin-group`, providing cluster-level permissions.
	- Added new `ClusterRole` named `kubeapps-admin`, granting specific permissions for resource management.

- **Bug Fixes**
	- None

- **Documentation**
	- None

- **Refactor**
	- None

- **Style**
	- None

- **Tests**
	- None

- **Chores**
	- None

- **Revert**
	- None

### CODE REVIEW

File: packages/system/keycloak-configure/templates/configure-kk.yaml [hunk 1]
```diff
@@ -225,3 +225,16 @@ spec:
...
     name: keycloakrealm-cozy
     kind: ClusterKeycloakRealm
+
+---
+
+apiVersion: v1.edp.epam.com/v1
+kind: KeycloakRealmGroup
+metadata:
+  name: cozystack-cluster-admin
+  namespace: cozy-dashboard
```
Comment (kvaps) [inline packages/system/keycloak-configure/templates/configure-kk.yaml]: Let's move it to cozy-system namespace

```suggestion
  namespace: cozy-system
```

File: packages/system/keycloak-configure/templates/rolebinding.yaml [hunk 2]
```diff
@@ -6,8 +6,40 @@ metadata:
...
+  name: kubeapps-admin
+  apiGroup: rbac.authorization.k8s.io
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: cozystack-cluster-admin-group
+  namespace: cozy-dashboard
```
Comment (kvaps) [inline packages/system/keycloak-configure/templates/rolebinding.yaml]: Agree with bot, makes no sense here
```suggestion
```

File: packages/system/keycloak-configure/templates/roles.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  - helmreleases
+  verbs:
+  - '*'
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: kubeapps-admin
+  namespace: cozy-public
```
Comment (kvaps) [inline packages/system/keycloak-configure/templates/roles.yaml:29]:
```suggestion
  namespace: cozy-dashboard
```

or `cozy-system` if you want to

`cozy-public` is the namespace intended to use for resoruces avialilable for all users

File: packages/system/keycloak-configure/templates/rolebinding.yaml [hunk 4]
```diff
@@ -6,8 +6,40 @@ metadata:
...
   kind: Group
   name: kubeapps-admin
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: kubeapps-admin
+  namespace: cozy-public
```
Comment (kvaps) [inline packages/system/keycloak-configure/templates/rolebinding.yaml:21]:
```suggestion
  namespace: cozy-dashboard
```

Comment (klinch0) [reply packages/system/keycloak-configure/templates/rolebinding.yaml:21]: its need for this role:

```
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  name: kubeapps-admin
  namespace: cozy-public
rules:
  - apiGroups: ["source.toolkit.fluxcd.io"]
    resources: ["helmrepositories"]
    verbs:
    - get
    - list
  - apiGroups: ["source.toolkit.fluxcd.io"]
    resources:
    - helmcharts
    verbs: ["*"]
```

**Review by kvaps — APPROVED** [review]
LGTM

**Pull request merged**